src/ralph/licences/models.py

```diff
@@ -191,8 +192,8 @@ class Licence(Regionalizable, AdminAbsoluteUrlMixin, BaseObject):
         help_text="Leave blank if this licence is perpetual",
     )
     order_no = models.CharField(max_length=50, null=True, blank=True)
-    price = models.DecimalField(
-        max_digits=10, decimal_places=2, default=0, null=True, blank=True,
+    price = MoneyField(
```

Comment:
Shouldn't the `price_currency` field be also reflected on the model?

Reply:
The field `price_currency` is automatically created by the library. `MoneyField` combines the two fields (`price` and `price_currency`) into one model.